### PR TITLE
Translation fetching fail

### PR DESCRIPTION
**Pull request by candrikos**
Add fault tolerance while fetching translations

    - Whenever fetching translations for a particular language fails,
      we need to keep on fetching translations of the rest of languages.
    - Update interval to be configurable
    - Increasing code coverage

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,6 +73,7 @@ USE_L10N = True
 # as found in the Transifex UI under your project
 TRANSIFEX_TOKEN = <project_token>  # used for pulling translations from Transifex
 TRANSIFEX_SECRET = <project_secret>  # used for pushing source content to Transifex
+TRANSIFEX_DAEMON_INTERVAL = <seconds> # used for defining the daemon running interval in seconds
```
Comment (sergafts) [inline README.md:76]: I'd prefer this to be something like `TRANSIFEX_SYNC_INTERVAL`, looks kinda clearer

File: transifex/native/daemon.py [hunk 2]
```diff
@@ -13,14 +13,16 @@ class DaemonicThread(threading.Thread):
     daemon = True
     should_exit = False
 
-    def start_daemon(self, interval=10):
+    def start_daemon(self, interval=None):
```
Comment (sergafts) [inline transifex/native/daemon.py:16]: I think you should use the INTERVAL setting here, and if it's not set, default back to `10*60`.

File: setup.py [hunk 3]
```diff
@@ -24,6 +25,6 @@
     ],
     url='https://github.com/transifex/transifex-python',
     install_requires=[
-        'pyseeyou', 'pytz', 'requests', 'six', 'click'
+        'pyseeyou', 'pytz', 'requests', 'six', 'click', 'parsimonious'
```
Comment (dbendilas) [inline setup.py:28]: Is `parsimonious` necessary for some reason?

Comment (candrikos) [reply setup.py:28]: Not you are right!

**Pull request merged**